lib/matplotlib/backends/backend_tkagg.py

```diff
@@ -609,6 +609,9 @@ def destroy(*args):
             self.window.update()
         else:
             self.canvas.draw_idle()
+        # Raise the new window.
+        self.canvas.manager.window.attributes('-topmost', 1)
```

Comment:
Long-term, it might make sense to have a standard API for requests to the window manager.  (We can't do more than make requests; each particular window manager has the final say.)  I'm reluctant to try to design this now, though, so I'm not sure it would be a good idea to add to our public API by putting this (and the raise() in qt) into methods.  It's debatable, and I have been on the fence about it.